package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@webrecorder/browsertrix-client",
+  "version": "1.0.0",
+  "description": "A JS Client for interacting with the backend of Browsertrix Cloud",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/webrecorder/browsertrix-client.git"
+  },
+  "author": "Webrecorder",
+  "license": "AGPL-3.0-or-later",
```

Comment:
I took this from browsertrix-cloud, I presume it's okay here too? (It may even be required?)

Reply:
Seems reasonable for the client to have the same license as the application!